src/test/java/org/folio/rest/api/RequestExpirationApiTest.java

```diff
@@ -66,11 +66,10 @@ public void checkIdsAfterEach() {
   }
 
   @Test
-  public void canExpireASingleOpenUnfilledRequest()
-    throws InterruptedException,
-    MalformedURLException,
-    TimeoutException,
-    ExecutionException {
+  public void canExpireASingleOpenUnfilledRequest() throws InterruptedException,
+    MalformedURLException, TimeoutException, ExecutionException {
+
+    stubTlrSettings(false);
```

Comment:
can we disable it by default in `@BeforeEach` to avoid adding this line to every test?